cypress/integration/api.ts

```diff
@@ -4,27 +4,85 @@ describe('API component', () => {
     cy.redirectUnpkgToLocalDistFolder('elements');
   });
 
-  it('loads correctly', () => {
-    loadZoomApiPage();
+  describe('Api page', () => {
+    it('loads api page correctly', () => {
+      loadZoomApiPage();
+      cy.findByText('https://api.zoom.us/v2').should('exist');
+      cy.findByRole('heading', { name: /API Key/ }).should('exist');
+      cy.findByText('API Description').should('exist');
+    });
   });
 
-  it('navigates via table of contents', () => {
-    loadZoomApiPage();
-    cy.findByText('groups').click();
-    cy.findByText('Create a group').click();
-    cy.findByRole('heading', { name: /Create a group/i }).should('exist');
+  describe('Operation page', () => {
+    it('loads operation docs correctly', () => {
+      cy.visit('/zoom-api/operations/groupCreate');
+      cy.findByRole('heading', { name: /Create a group/i }).should('exist');
+      cy.findByRole('heading', { name: /Request/i }).should('exist');
+      cy.findByRole('heading', { name: /post \/groups/i }).should('exist');
+      cy.findByRole('heading', { name: /Response/i }).should('exist');
+    });
+
+    it('does not break on select dropdown', () => {
+      loadZoomApiPage();
+      cy.visit('/zoom-api/operations/groups');
+      cy.findByRole('button', { name: /request sample/i }).click();
+      cy.findByText('Obj-C').should('exist');
+    });
+
+    it('displays request sample correctly', () => {
+      cy.visit('/zoom-api/operations/userCreate');
+      cy.findByLabelText(
+        'curl --request POST \\ --url \'https://api.zoom.us/v2/users?access_token=\' \\ --header \'Content-Type: application/json\' \\ --data \'{ "action": "create", "user_info": { "email": "string", "first_name": "string", "last_name": "string", "password": "string", "type": 1 } }\'',
```

Comment:
This may be too specific to the implementation. I'm thinking: There's about a million ways to send the same request via `curl`, but enough if you just think about the order of arguments. This will break even when the result is still perfectly correct. This may be a compromise we accept of course, but I'm thinking that as this is an e2e that is supposed to be high level (and we already have a unit test for the same topic) it may be enough here to assert some basics that are less fragile

Reply:
That's true. I simplified it